### PR TITLE
tests(ServiceCreateModal): Integration tests for Single Container

### PR DESCRIPTION
**Pull request by MatApple**
Add's full integration test coverage for Service Create Modal (Single Container) and closes https://jira.mesosphere.com/browse/DCOS-14461.

Although ~1000 lines added, they're all integration tests and with their nested contexts, it makes sense to land as a single PR to avoid rebase time and loss of context when reviewing. Please let me know if you disagree.

At some point we should consider splitting this file, perhaps once all Service Create integration tests have been committed to master.

**Checklist**
- [x] Did you add a JIRA issue in a commit message or as part of the branch name?
- [ ] Did you add new unit tests? (N/A)
- [x] Did you add new integration tests?
- [ ] If this is a regression, did you write a test to catch this in the future? (N/A)

### CODE REVIEW

File: tests/pages/services/ServiceFormModal-cy.js [hunk 1]
```diff
@@ -431,16 +600,865 @@ describe('Service Form Modal', function () {
...
+
+    context('Service: Volumes', function () {
+
+      beforeEach(function () {
+        cy.get('.menu-tabbed-item')
+          .contains('Volumes')
+          .click();
+
+        // Alias tab view for cached lookups
+        cy.get('.menu-tabbed-view').as('tabView');
```
Comment (weblancaster) [inline tests/pages/services/ServiceFormModal-cy.js]: Nice!

**Comment by weblancaster** [comment]
LGTM 👍

**Comment by weblancaster** [comment]
retest this please.

**Comment by nLight** [comment]
For me that one is failing, text field is empty on the video:
1) Service Versions Configuration Tab opens correct edit modal of the selected service version:
     CypressError: Timed out retrying: Expected to find content: 'sleep 1000' within the element: <textarea
.form-control> but never did.

Also these are failing:
2) Units Tab [0e2] Filters [0e3] "before each" hook for "renders filter buttons [0e4]":
3) Units Tab [0e2] Unit Detail Page [0e9] "before each" hook for "renders unit title [0ea]":
4) Units Tab [0e2] Unit Node Side Panel [0ef] "before each" hook for "renders health check title [0ei]":

because of `Error: Module ResourceTableUtil does not exist.` which could be my fault messing around with (half reviewed/merged) foundation

**Comment by weblancaster** [comment]
@nLight I would suggest always test fresh/as closely possible to the PR environment. That being said, I have run locally and I had no issues.

**Comment by nLight** [comment]
@weblancaster I tested with master branch on plugins, if there's something that could bring me even closer to the original env, would please guide me a little bit?

**Comment by weblancaster** [comment]
Perhaps I'm the one testing wrong. I pulled the branch and ran the tests without merging/rebasing to my local master @nLight

**Comment by nLight** [comment]
@weblancaster I did the same. By 'my fault' I mean that there're bits I've changed as parts of the foundation-ui extraction that could cause `ResourceTableUtil` not found error. I never rebase a PR right away. Though I try to do so if my first attempt to test it fails.

**Comment by nLight** [comment]
Retested and it works if you rebase it on the master. This is because the foundation PR for plugins has been merged and unless you rebase this branch it will be failing with the latest plugins master

**Comment by nLight** [comment]
⚠️ I didn't force push the rebase though

**Comment by mlunoe** [comment]
I get the same error, will rebase and push if that fixes it. It has been ok'ed by @MatApple himself 👍

**Comment by mlunoe** [comment]
Let's merge when tests pass :shipit:

**Pull request merged**